cache.go

```diff
@@ -67,3 +68,10 @@ func GetPostsCache(userID int64) *[]PublicPost {
 	}
 	return pci.Posts
 }
+
+func cacheControl(next http.Handler) http.Handler {
```

Comment:
Since `cache.go` is meant more for internal application caches, I'd suggest moving this to a place where we have other HTTP middleware, like `handle.go`.

Reply:
Sure thing! I moved it in [this commit](https://github.com/writeas/writefreely/pull/336/commits/1b1d3064c9064ca76ee629d07ff79fbdaffd31ca#diff-fe997d9c2c36e76b19925e7f5fad6e48).